### ISSUE TITLE
TESTS: Drop test with 'transparent' futures

### ISSUE THREAD

**Issue by HenrikBengtsson** [nhejazi/haldensify#19]
Hi.  As part of taking the **future** package to the next level, I'm narrowing down what futures may do.  This is mostly backward compatible, but one thing that needs to go is the support for `future(..., local = FALSE)`.  Most of this has already been done, cf. https://github.com/HenrikBengtsson/future/issues/382.  However, it is still used internally by `transparent` futures, which I'll keep supporting for a while, but they should only be used for debugging and troubleshooting purposes. They should never for production use  The reason for this is that `local = FALSE` introduces behavior that is an outlier in the future framework.   (This will be clarified further in the next release of **future**).

I spotted that **haldensify** uses 'transparent' futures in two of its package tests:

https://github.com/nhejazi/haldensify/blob/337a443bd18cf53778eca2dd05f8ea70c5433c7f/tests/testthat/test-density_standard.R#L6

https://github.com/nhejazi/haldensify/blob/337a443bd18cf53778eca2dd05f8ea70c5433c7f/tests/testthat/test-density_weights.R#L6

Could you please replace these to test with `sequential` futures, or even better, just rely on the default.  By relying on the default, the packages will also be validated against other backends that I test against when I run my **future** revdep checks, e.g. `R_FUTURE_PLAN=multisession`.

PS. I see that you used to test with `sequential` but then switched to `transparent`. What was the reason for doing that?

**Comment by nhejazi** [comment]
Thanks @HenrikBengtsson --- in an unrelated package (private repo), I actually ran into the relatively recent change that causes issues with `plan(transparent)`. I'll go ahead and drop the specification of `plan(transparent)` from these tests so that the default gets used in your revdep checks. You're right, I used to use specify `plan(sequential)` in tests but this became a pain for debugging purposes (if I recall correctly, `plan(transparent)` plays well with `browser()` whereas `plan(sequential)` doesn't), so I've lately been using `plan(transparent)`.
